core/src/main/java/net/adoptopenjdk/icedteaweb/jnlp/element/information/InformationDesc.java

```diff
@@ -255,11 +267,11 @@ protected Object getItem(Object key) {
      * @param key key to find item
      * @return all items matching the specified key.
      */
-    public List<Object> getItems(Object key) {
+    public List<Object> getItems(final Object key) {
```

Comment:
`addItem()` only allows String as a key. Therefore we should also restrict the parameter of `getItems()` to be a String and not an Object

Reply:
done